packages/core/src/test/codewhisperer/commands/startSecurityScanOld.test.ts

```diff
@@ -0,0 +1,271 @@
+const mockCreateCodeScanResponse = {
+    $response: {
+        data: {
+            jobId: 'jobId',
+            status: 'Pending',
+        },
+        requestId: 'requestId',
+        hasNextPage: () => false,
+        error: undefined,
+        nextPage: () => undefined,
+        redirectCount: 0,
+        retryCount: 0,
+        httpResponse: new HttpResponse(),
+    },
+    jobId: 'jobId',
+    status: 'Pending',
+}
+
+const mockCreateUploadUrlResponse = {
+    $response: {
+        data: {
+            uploadId: 'uploadId',
+            uploadUrl: 'uploadUrl',
+        },
+        requestId: 'requestId',
+        hasNextPage: () => false,
+        error: undefined,
+        nextPage: () => undefined,
+        redirectCount: 0,
+        retryCount: 0,
+        httpResponse: new HttpResponse(),
+    },
+    uploadId: 'uploadId',
+    uploadUrl: 'https://test.com',
+}
+
+const mockGetCodeScanResponse = {
+    $response: {
+        data: {
+            status: 'Completed',
+        },
+        requestId: 'requestId',
+        hasNextPage: () => false,
+        error: undefined,
+        nextPage: () => undefined,
+        redirectCount: 0,
+        retryCount: 0,
+        httpResponse: new HttpResponse(),
+    },
+    status: 'Completed',
+}
+
+const mockCodeScanFindings = JSON.stringify([
+    {
+        filePath: '/workspaceFolder/python3.7-plain-sam-app/hello_world/app.py',
+        startLine: 1,
+        endLine: 1,
+        title: 'title',
+        description: {
+            text: 'text',
+            markdown: 'markdown',
+        },
+        detectorId: 'detectorId',
+        detectorName: 'detectorName',
+        findingId: 'findingId',
+        relatedVulnerabilities: [],
+        severity: 'High',
+        remediation: {
+            recommendation: {
+                text: 'text',
+                url: 'url',
+            },
+            suggestedFixes: [],
+        },
+    } satisfies model.RawCodeScanIssue,
+])
+
+const mockListCodeScanFindingsResponse = {
+    $response: {
+        data: {
+            codeScanFindings: mockCodeScanFindings,
+        },
+        requestId: 'requestId',
+        hasNextPage: () => false,
+        error: undefined,
+        nextPage: () => undefined,
+        redirectCount: 0,
+        retryCount: 0,
+        httpResponse: new HttpResponse(),
+    },
+    codeScanFindings: mockCodeScanFindings,
+}
+
+let extensionContext: FakeExtensionContext
+let mockSecurityPanelViewProvider: SecurityPanelViewProvider
+let appRoot: string
+let appCodePath: string
+let editor: vscode.TextEditor
+
+describe('startSecurityScanOld', function () {
+    const workspaceFolder = getTestWorkspaceFolder()
+
+    beforeEach(async function () {
+        extensionContext = await FakeExtensionContext.create()
+        mockSecurityPanelViewProvider = new SecurityPanelViewProvider(extensionContext)
+        appRoot = join(workspaceFolder, 'python3.7-plain-sam-app')
+        appCodePath = join(appRoot, 'hello_world', 'app.py')
```

Comment:
there is never a reason to use join() on constant strings. and it's less readable.

```suggestion
        appCodePath = join(appRoot, 'hello_world/app.py')
```